pages/index.html

```diff
@@ -1,12 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Asha's Personal Portfolio</title>
+    <link rel="stylesheet" href="/styles/style.css">
```

Comment:
If you move index.html to the project root, the path to `style.css` would be:

```suggestion
    <link rel="stylesheet" href="./styles/style.css">
```